test/camera/camera_test.py

```diff
@@ -0,0 +1,391 @@
+        camera.pan(180)
+
+        self.assertEqual(camera._current_pan_angle, 180)
+
+
+    def test_pan_left_absolute_beyond_bound(self):
+        camera = Camera()
+        camera.pan(181)
+
+        self.assertEqual(camera._current_pan_angle, 180)
+
+
+    def test_pan_right_absolute(self):
+        camera = Camera()
+        camera.pan(60)
+
+        self.assertEqual(camera._current_pan_angle, 60)
+
+
+    def test_pan_right_absolute_at_bound(self):
+        camera = Camera()
+        camera.pan(0)
+
+        self.assertEqual(camera._current_pan_angle, 0)
+
+
+    def test_pan_right_absolute_beyond_bound(self):
+        camera = Camera()
+        camera.pan(-1)
+
+        self.assertEqual(camera._current_pan_angle, 0)
+
+
+    def test_pan_left_relative(self):
+        camera = Camera(camera_type=RELATIVE)
+        camera.pan(30)
+
+        self.assertEqual(camera._current_pan_angle, 120)
+
+
+    def test_pan_left_relative_at_bound(self):
+        camera = Camera(camera_type=RELATIVE)
+        camera.pan(90)
+
+        self.assertEqual(camera._current_pan_angle, 180)
+
+
+    def test_pan_left_relative_beyond_bound(self):
+        camera = Camera(camera_type=RELATIVE)
+        camera.pan(91)
+
+        self.assertEqual(camera._current_pan_angle, 180)
+
+
+    def test_pan_right_relative(self):
+        camera = Camera(camera_type=RELATIVE)
+        camera.pan(-30)
+
+        self.assertEqual(camera._current_pan_angle, 60)
+
+
+    def test_pan_right_relative_at_bound(self):
+        camera = Camera(camera_type=RELATIVE)
+        camera.pan(-90)
+
+        self.assertEqual(camera._current_pan_angle, 0)
+
+
+    def test_pan_right_relative_beyond_bound(self):
+        camera = Camera(camera_type=RELATIVE)
+        camera.pan(-91)
+
+        self.assertEqual(camera._current_pan_angle, 0)
+
+
+    def test_tilt_up_absolute(self):
+        camera = Camera()
+        camera.tilt(90)
+
+        self.assertEqual(camera._current_tilt_angle, 90)
+
+
+    def test_tilt_up_absolute_at_bound(self):
+        camera = Camera()
+        camera.tilt(150)
+
+        self.assertEqual(camera._current_tilt_angle, 150)
+
+
+    def test_tilt_up_absolute_beyond_bound(self):
+        camera = Camera()
+        camera.tilt(151)
+
+        self.assertEqual(camera._current_tilt_angle, 150)
+
+
+    def test_tilt_down_absolute(self):
+        camera = Camera()
+        camera.tilt(90)
+        camera.tilt(60)
+
+        self.assertEqual(camera._current_tilt_angle, 60)
+
+
+    def test_tilt_down_absolute_at_bound(self):
+        camera = Camera()
+        camera.tilt(90)
+        camera.tilt(0)
+
+        self.assertEqual(camera._current_tilt_angle, 0)
+
+
+    def test_tilt_down_absolute_beyond_bound(self):
+        camera = Camera()
+        camera.tilt(90)
+        camera.tilt(-1)
+
+        self.assertEqual(camera._current_tilt_angle, 0)
+
+
+    def test_tilt_up_relative(self):
+        camera = Camera(camera_type=RELATIVE)
+        camera.tilt(90)
```

Comment:
You don't need to tilt up twice on this one. Do you? can't you just assert that it's at 90?